class/Patchwork/Utf8.php

```diff
@@ -555,7 +555,7 @@ static function substr_replace($s, $replace, $start, $len = 2147483647)
     static function ucfirst($s)
     {
         $c = iconv_substr($s, 0, 1, 'UTF-8');
-        return static::ucwords($c) . substr($s, strlen($c));
+        return static::strtoupper($c) . substr($s, strlen($c));
```

Comment:
We are only converting one character here, so shouldn't upper case be the same as title case? strtoupper should be faster than ucwords for a single character. Also, this now mirrors lcwords, which uses strtolower.